client/verta/docs/api/api/experimentrun.rst

```diff
@@ -141,9 +128,7 @@ Deprecated
 .. automethod:: ExperimentRun.log_model_for_deployment
 .. automethod:: ExperimentRun.log_modules
 
-.. _experiment-run-versioning:
+Miscellaneous
+-------------
 
-Versioning
-^^^^^^^^^^
-.. automethod:: ExperimentRun.log_commit
-.. automethod:: ExperimentRun.get_commit
```

Comment:
@mvartakAtVerta Was this removed intentionally?

Reply:
nope, my bad